src/scripts/data_processing/generate_data/generate.py

```diff
@@ -93,64 +95,18 @@ def get_arg_parser(cls, parser=None):
             type=str,
             help="Output data location (directory)",
         )
-
-        return parser
-
-    def generate_classification(self, args):
-        total_samples = (
-                args.train_samples + args.test_samples + args.inferencing_samples
-        )
-
-        X, y = make_classification(
-            n_samples=total_samples,
-            n_features=args.n_features,
-            n_informative=args.n_informative,
-            n_redundant=args.n_redundant,
-            random_state=args.random_state,
+        group_o.add_argument(
+            "--output_header",
+            required=True,
```

Comment:
could you make it required in the yaml spec but not in the script? this way, if someone runs this script locally for some reason they don't have to provide this arugment?

Reply:
done.